README.md

```diff
@@ -173,6 +173,9 @@ Releases for Ollama are maintained on [GitHub](https://github.com/ollama/ollama/
 
 ### NVIDIA CUDA
 
+> [!TIP]
+> Running the commands below is not enough to ensure that the latest version of NVIDIA CUDA is install. Double check there is not a minor or major version increase by inspecting the contents of the [CUDA repository](https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2404/x86_64/).
+
```

Comment:
```suggestion
> [!TIP]
> Running the commands below is not enough to ensure that the latest version of NVIDIA CUDA is installed. Double check there is not a minor or major version increase by inspecting the contents of the [CUDA repository](https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2404/x86_64/).

```

do you want to make reference to this instead https://gitlab.com/nvidia/container-images/cuda/-/tree/master/dist?

Reply:
Sure thing, let me update that!